go/runtime/bundle/component.go

```diff
@@ -255,6 +260,35 @@ func (r *TDXResources) Validate() error {
 	if r.CPUCount < 1 {
 		return fmt.Errorf("vCPU count must be at least 1")
 	}
+	if r.GPU != nil {
+		err := r.GPU.Validate()
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+const (
+	// GPUKindNvidiaH100 is the NVIDIA H100 GPU kind.
+	GPUKindNvidiaH100 = "nvidia-h100"
```

Comment:
Can we remove `Kind`?

Reply:
Yes, we could just remove it for now.

Reply:
I should have been more specific. What I meant is whether we can remove `Kind` from the name and change it to `GPUNvidiaH100` or `gpu.NvidiaH100` instead.